pom.xml

```diff
@@ -72,10 +73,29 @@ SOFTWARE.
   </scm>
   <dependencyManagement>
     <dependencies>
+       <dependency>
+        <groupId>io.vertx</groupId>
+        <artifactId>vertx-dependencies</artifactId>
+        <version>${vertx.version}</version>
+        <type>pom</type>
```

Comment:
@victornoel why did you change this dependency?

Reply:
@g4s8 I didn't change them, I changed how they were declared so we get all the dependencies management for all the vertx dependency at once instead of declaring them one by one. This is standard practice for handling this kind of dependencies, see https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies for details